apis/core.oam.dev/v1beta1/applicationrevision_types.go

```diff
@@ -56,7 +56,7 @@ type ApplicationRevisionSpec struct {
 
 // ApplicationRevision is the Schema for the ApplicationRevision API
 // +kubebuilder:storageversion
-// +kubebuilder:resource:categories={oam},shortName=apprev;revisions
+// +kubebuilder:resource:categories={oam},shortName=apprev;revision
```

Comment:
same concern and `apprev` is enough I think

Reply:
Done